python/ray/experimental/sgd/mnist_example.py

```diff
@@ -55,8 +55,11 @@ def __init__(self):
         if error:
             raise ValueError("Failed to import data", error)
 
+    def build(self):
```

Comment:
Do we need build()? Seems it can be unified with the constructor to simplify.

Reply:
I move the graph constructing process out of the constructor because I want to make the constructor simpler.